### DISCUSSION TITLE
¿Como cambiar el Amount en el cardForm?

### DISCUSSION THREAD

**Discussion by emilioHayes** [mercadopago/sdk-js#96] [Q&A] [answered]
En la inicialización del cardForm se tiene que definir el Amount, ¿ Es posible cambiar la cantidad de forma dinámica sin tener que recargar la vista actual?
```
 const cardForm = mp.cardForm({
      amount: "100.5", <--- ???
      iframe: true,
      form: {
        id: "form-checkout",
        cardNumber: {
          id: "form-checkout__cardNumber",
          placeholder: "Numero de tarjeta",
        }....
```

**Comment by lucmantovani** [comment, marked as answer]
Hola, @emilioHayes!

En este momento no hay manera de cambiar el `amount` de forma dinámica
Lo único que se puede hacer es enviar un valor diferente del definido en la inicialización del cardForm para el backend
